lacci/lib/shoes/drawables/button.rb

```diff
@@ -22,12 +22,21 @@ def initialize(text, width: nil, height: nil, top: nil, left: nil, color: nil, p
         @block&.call
       end
 
+      bind_self_event("hover") do
+        @log.debug("Button hovered, calling handler") if @block
```

Comment:
The debug should probably not stay in the final version.

Reply:
hm any reason for that? when we are doing log,debug for click? and why not for hover?

Reply:
ooo okay